client/layout/index.jsx

```diff
@@ -143,14 +142,11 @@ const Layout = createReactClass( {
 						{ this.props.primary }
 					</div>
 				</div>
-				{ config.isEnabled( 'i18n/community-translator' ) && isCommunityTranslatorEnabled() ? (
-					<AsyncLoad require="components/community-translator" />
-				) : (
-					<TranslatorLauncher
-						isEnabled={ translator.isEnabled() }
-						isActive={ translator.isActivated() }
-					/>
-				) }
+				{ config.isEnabled( 'i18n/community-translator' ) &&
+					isCommunityTranslatorEnabled() && (
+						<AsyncLoad require="components/community-translator" />
+					) }
+				<TranslatorLauncher />
```

Comment:
Note that it was intentional that if `i18n/community-translator` is enabled then the launcher won't be docked here. `components/community-translator` is a revamped version of the whole community translator and it doesn't rely on `<TranslatorLauncher />` to work. Sorry that I didn't catch this beforehand, and we should have stated it clearer in the first place :)

Reply:
@southp, @ramonjd, thank you for telling me about `i18n/community-translator`. I've added the logic back in 2a3bf2b89e9bbb186791b61bbdee1b9ef84a13d1. A minor difference is that I had to move `isCommunityTranslatorEnabled()` out of the way of `<TranslatorLauncher />`. 

This is because during the first rendering of `<Layout />`, `isCommunityTranslatorEnabled()` is most likely resolved to `false`, preventing `<TranslatorLauncher />` from the first mount, and in terms stopping the change handler registrations of i18n and userSettings. If those handlers are not registered, then there's no way to trigger the `<TranslatorLauncher />` to render later.